src/Microsoft.DotNet.Darc/src/DarcLib/IBarClient.cs

```diff
@@ -96,6 +96,13 @@ public interface IBarClient
         /// <returns>Subscription just triggered.</returns>
         Task<Subscription> TriggerSubscriptionAsync(Guid subscriptionId);
 
+        /// <summary>
+        /// Trigger a subscription by ID and source build identity.
```

Comment:
[nit]

```suggestion
        /// Trigger a subscription by ID and source build id
```

Just to be consistent with other summaries?